tools/firtool/firtool.cpp

```diff
@@ -329,39 +249,17 @@ static cl::opt<bool> exportModuleHierarchy(
     cl::desc("Export module and instance hierarchy as JSON"), cl::init(false),
     cl::cat(mainCategory));
 
-static cl::opt<bool>
-    disableCheckCombCycles("disable-check-comb-cycles",
-                           cl::desc("Disable the CheckCombCycles pass"),
-                           cl::init(false), cl::Hidden, cl::cat(mainCategory));
```

Comment:
This type of thing _could_ have a use. However, let's just remove it and we can add it back in later if needed.